pkg/diff/diff.go

```diff
@@ -207,16 +213,12 @@ func applyDefaultValues(result *typed.TypedValue) ([]byte, error) {
 	return mergedBytes, nil
 }
 
-func buildDiffResult(mergedBytes []byte, live *unstructured.Unstructured) (*DiffResult, error) {
+func buildDiffResult(predictedBytes []byte, liveBytes []byte) (*DiffResult, error) {
```

Comment:
Could this just be incorporated into `structuredMergeDiff`, since it's trivial at this point?

Reply:
good idea. done!